pkg/operator/latencyprofilecontroller/controller.go

```diff
@@ -82,20 +94,20 @@ func (c *LatencyProfileController) sync(ctx context.Context, syncCtx factory.Syn
 	// Collect the current latency profile
 	configNodeObj, err := c.configNodeLister.Get("cluster")
 
-	// if config/v1/node/cluster object is not found this controller should do nothing
-	if err != nil && !apierrors.IsNotFound(err) {
-		return err
-	}
-
-	// in case of empty workerlatency profile, set status Complete=False and pre-empt sync
-	if apierrors.IsNotFound(err) || configNodeObj.Spec.WorkerLatencyProfile == "" {
-		_, err = c.updateStatus(
-			ctx,
-			false, true, // not progressing, complete=True
-			reasonLatencyProfileEmpty,
-			"latency profile not set on cluster",
-		)
-		return err
+	// if config/v1/node/cluster object is not found
+	if err != nil {
+		// in case of config node object doesn't exist
+		if apierrors.IsNotFound(err) {
```

Comment:
Refactored the if branch(es) to make them almost identical in pattern.

```go
        // if config/v1/node/cluster object is not found
	if err != nil && !apierrors.IsNotFound(err) {
		return err
	}

	// in case of config node object doesn't exist
	if apierrors.IsNotFound(err) {
		_, err = c.updateStatus(
			ctx,
			false, true, // not progressing, complete=True
			reasonLatencyProfileEmpty,
			"nodes.config.openshift.io/cluster object was not found on the cluster",
		)
		return err
	}
```
How about this (updated code)?

Reply:
the branching standard looks good now, but please double check https://github.com/openshift/library-go/pull/1372#discussion_r899319804

Reply:
Thanks.